server/helm/rayyan-store/value-overrides/local.yaml

```
@@ -0,0 +1,2 @@
+secrets:
+  RAYYAN_STORE_DATABASE_URLS: [DUMMY]
```

Better write in native YAML?
```suggestion
  RAYYAN_STORE_DATABASE_URLS:
  - DUMMY
```